modules/data.atmosphere/R/download.CRUNCEP_Global.R

```diff
@@ -0,0 +1,20 @@
+download.CRUNCEP <- function(outfolder, start_year, end_year, lat, lon, overwrite=FALSE, verbose=FALSE, ...){
+  
+  require(PEcAn.utils)
+  require(lubridate)
+#  start_date <- as.POSIXlt(start_date, tz = "GMT")
+#  end_date <- as.POSIXlt(end_date, tz = "GMT")
+#  start_year <- year(start_date)
+#   end_year   <- year(end_date)
+  
+# start_year = 2001
+#  end_year = 2005
+#  lat = 45.0
+#  lon = -90.2
+  ylist <- seq(start_year,end_year,by=1)
+    for (year in ylist) 
+      {
+      system(paste('/usr/local/anaconda2/bin/python ~/pecan/modules/data.atmosphere/python/download_Global_MsTMIP_CRUNCEP.py ',lat,lon,year))
```

Comment:
That makes sense. I think it will take some more python voodoo to get the VM to actually work with the code, but you’re right the R wrapper is not the place for those kluges. I’ll have James look into the system.file command. 
-------------------------------------------------
Ankur R Desai, Associate Professor
  University of Wisconsin - Madison, Atmospheric and Oceanic Sciences
  http://flux.aos.wisc.edu <http://flux.aos.wisc.edu/>  desai@aos.wisc.edu <mailto:desai@aos.wisc.edu> 
O: +1-608-520-0305 / M: +1-608-218-4208



> On Dec 6, 2015, at 7:24 PM, Michael Dietze <notifications@github.com> wrote:
> 
> In modules/data.atmosphere/R/download.CRUNCEP_Global.R <https://github.com/PecanProject/pecan/pull/696#discussion_r46780546>:
> 
> > +  
> > +  require(PEcAn.utils)
> > +  require(lubridate)
> > +#  start_date <- as.POSIXlt(start_date, tz = "GMT")
> > +#  end_date <- as.POSIXlt(end_date, tz = "GMT")
> > +#  start_year <- year(start_date)
> > +#   end_year   <- year(end_date)
> > +  
> > +# start_year = 2001
> > +#  end_year = 2005
> > +#  lat = 45.0
> > +#  lon = -90.2
> > +  ylist <- seq(start_year,end_year,by=1)
> > +    for (year in ylist) 
> > +      {
> > +      system(paste('/usr/local/anaconda2/bin/python ~/pecan/modules/data.atmosphere/python/download_Global_MsTMIP_CRUNCEP.py ',lat,lon,year))
> Ok, so you're going to need to fix the paths here, which are local to your machine. For the first, it's probably safe to assume that folks have python in their $PATH and thus you shouldn't need the path. For the second, you need to use R's system.file command to retrieve the system-specific location of your script within the PEcAn.data.atmosphere package.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/PecanProject/pecan/pull/696/files#r46780546>.
>